docs/cep/error-handling/error-handling-wait-and-retry.md

```diff
@@ -5,54 +5,37 @@ title: Wait and Retry Example
+In a scenario where connections to external systems are intermittent, the `OnError.action='wait'` property enables a sink to temporarily hold event publication until the connection is re-established. This behavior introduces back pressure, preventing new events from being consumed until the issue is resolved.
 
-Here is a simple example to publish abnormal glucose reading events to an unavailable HTTP endpoint. The error is handled by `wait and retry` mode.
+Here is an example demonstrating how to configure a sink to handle connection errors by using a wait and retry strategy:
 
 ```sql
--- Defines `GlucoseReadingStream` stream which contains events related to glucose readings.
-CREATE STREAM GlucoseReadingStream (locationRoom string,
-    locationBed string, timeStamp string, sensorID long,
-    patientFirstName string, patientLastName string,
-    sensorValue double);
+-- Defines the stream for handling glucose readings
+CREATE STREAM GlucoseReadingStream (locationRoom string, locationBed string, timeStamp string, sensorID long, patientFirstName string, patientLastName string, sensorValue double);
 
--- If `HTTP` endpoint is unavailable then threads who bring events via `AbnormalGlucoseReadingStream` wait in `back-off and re-trying` mode.
-
-@OnError(action='wait')
-CREATE SINK AbnormalGlucoseReadingStream WITH (type = 'http', OnError.action="wait", publisher.url = "http://localhost:8080/logger", method = "POST", map.type = 'json') (timeStampInLong long, locationRoom string, locationBed string, sensorID long, patientFullName string, sensorReadingValue double);
+-- Sink configuration with the OnError.action='wait' property for wait and retry strategy
+CREATE SINK AbnormalGlucoseReadingStream WITH (type = 'http', OnError.action='wait', publisher.url = "http://localhost:8080/logger", method = "POST", map.type = 'json') (timeStampInLong long, locationRoom string, locationBed string, sensorID long, patientFullName string, sensorReadingValue double);
 
 @info(name='abnormal-reading-identifier')
 insert into AbnormalGlucoseReadingStream
-select math:parseLong(timeStamp) as timeStampInLong,
-    locationRoom, locationBed, sensorID,
-    -- Concatenate string attributes `patientFirstName` and `patientLastName`
-    str:concat(patientFirstName, " ", patientLastName) as patientFullName, sensorValue as sensorReadingValue
--- Identifies the abnormal glucose reading if `sensorValue > 220`
+select math:parseLong(timeStamp) as timeStampInLong, locationRoom, locationBed, sensorID, str:concat(patientFirstName, " ", patientLastName) as patientFullName, sensorValue as sensorReadingValue
 from GlucoseReadingStream[sensorValue > 220];
 ```
 
 ## Prerequisites
 
-Download the mock logger service from [here](https://github.com/mohanvive/siddhi-mock-services/releases/download/v2.0.0/logservice-1.0.0.jar).
+Ensure that the mock logger service is available for download as previously specified.
```

Comment:
@diana-macrometa / @StoyanTc - Do we have a `mock logger service` to download?

Reply:
Not that I am aware of. Sorry, that should not have been removed. Fixed.